apps/member/src/components/group/ActivityConfigTableSection/ActivityConfigTableSection.tsx

```diff
@@ -41,7 +41,9 @@ const ActivityConfigTableSection = ({
           <Table.Row key={board.id}>
             <Table.Cell>{index + 1}</Table.Cell>
             <Table.Cell className="truncate">{board.title}</Table.Cell>
-            <Table.Cell>{formattedDate(board.updatedAt)}</Table.Cell>
+            <Table.Cell>
+              {formattedDate(toKoreaISOString(board.updatedAt || ''))}
```

Comment:
```suggestion
              {formattedDate(toKoreaISOString(board.updatedAt ?? ''))}
```
```suggestion
              {formattedDate(toKoreaISOString(board.updatedAt))}
```

`toKoreaISOString` 내부에서 예외처리를 하는 건 어떨까요??